sway-book/book.toml

```diff
@@ -0,0 +1,18 @@
+[book]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+language = "en"
+multilingual = false
+src = "src"
+title = "The Sway Book"
+
+[output.html]
+git-repository-url = "https://github.com/FuelLabs/sway"
+
+[preprocessor.forc-documenter]
```

Comment:
```suggestion
```

We can remove the `forc-documenter` processor here since we probably are not documenting `forc` commands here(?)